Libraries/src/Amazon.Lambda.AspNetCoreServer/Internal/EncodingOptions.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+
+namespace Amazon.Lambda.AspNetCoreServer.Internal
```

Comment:
Seems like the `EncodingOptions` and interface should be in `Amazon.Lambda.AspNetCoreServer.Hosting`. They are basically metadata of the `HostingOptions`.